query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/ids/byoid.rs

```diff
@@ -169,7 +171,7 @@ mod byoid {
         );
 
         let error_target = match runner.connector_version() {
-            ConnectorVersion::MySql(_) => "constraint: `PRIMARY`",
+            ConnectorVersion::MySql(_) | ConnectorVersion::TiDB => "constraint: `PRIMARY`",
```

Comment:
Any idea why Vitess does not have to be listed here? Is it maybe set up a bit differently so it automatically applies here?

Reply:
@Mini256 ?

Reply:
Vitess will match the next condition value:

https://github.com/prisma/prisma-engines/blob/49ab8b879e4778a76fe9bfa2c85c8e10afd27b10/query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/ids/byoid.rs#L175

Reply:
Ahhh, Vitess does not provide the affected constraint, I remember (or we are not parsing it correctly).